README.md

```diff
@@ -84,6 +83,45 @@ Here's the whole `config.pbtxt` we use for the `ResizePipeline` example:
         }
         ]
 
+## Autoserialization
+
+When using DALI Backend in Triton, user has to provide a DALI model in the Model Repository.
+A canonical way of expressing a model is to include a serialized DALI model file there and
+naming the file properly (``model.dali`` by default). The issue that arises from storing model
+in a serialized file is that, after serialization, the model is obscure and almost impossible
+to read anymore. Autoserialization feature allows user to express the model in Python code in
+the model repository. With the Python-defined model, DALI Backend uses internal serialization
+mechanism and exempts user from manual serialization.
+
+To use the autoserialization feature, user needs to put a Python-definition of the DALI pipeline
+inside the model file (``model.dali`` by default, but the default file name can be configured
+in the ``config.pbtxt``). Such pipeline definition has to be decorated with ``@autoserialize``,
+e.g.:
+
+    import nvidia.dali as dali
+
+    @dali.triton.autoserialize
+    @dali.pipeline_def(batch_size=3, num_threads=1, device_id=0)
+    def pipe():
+        '''
+        An identity pipeline with autoserialization enabled
+        '''
+        data = dali.fn.external_source(device="cpu", name="DALI_INPUT_0")
+        return data
+
+
+Proper DALI pipeline definition in Python, together with autoserialization, shall meet the
+following conditions:
+1. Only a ``pipeline_def`` can be decorated with ``autoserialize``.
+2. Only one pipeline definition may be decorated with ``autoserialize`` in a given model version.
+
+While loading a model file, DALI Backend follows the precedence:
+1. First, DALI Backend tries to load a serialized model from the user-specified model location (``model.dali`` by default);
+2. If the previous fails, DALI Backend tries to load and autoserialize a Python pipeline
+definition from the user-specified model location. **Important**: Due to Python limitations, name of such file shall end with `.py`, e.g. `model.py`;
```

Comment:
This looks like a limitation that should be lifted by us.

Reply:
I mean, we have it by default in `model.dali` and according to this we look there for a non-serialized Python code, but it is error to try to read a file that doesn't end with .py. I'm confused.

Reply:
"Default location" means, that the user specified it in the model configuration:
```
default_model_filename: "mymodel.py"
```
It's just that, by default `default_model_filename: "model.dali"`. Basically the takeaway is: If you have a serialized model, call it `model.dali`. If you have a python definition - call it `dali.py`. Probably I should add such sentence there...

Reply:
Added